src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBlock.cs

```diff
@@ -54,5 +56,14 @@ public void Check_Text_When_Having_Inline_Text_In_Span()
 			Assert.AreEqual("did", ((Run)((Italic)inlines[1]).Inlines.Single()).Text);
 			Assert.AreEqual(" my text go?", ((Run)inlines[2]).Text);
 		}
+
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_Null_FontFamily()
+		{
+			var SUT = new TextBlock { Text = "Some text", FontFamily = null };
```

Comment:
UWP doesn't allow null `FontFamily` in the first place.

I'm thinking of introducing a `FrameworkPropertyMetadataOptions.DisallowNull` member and gradually applying it to properties that shouldn't allow null. It's going to be a breaking change though.

Reply:
It's modified this way because it's an important breaking change.